studio/components/interfaces/TableGridEditor/TableGridEditor.tsx

```diff
@@ -62,7 +70,8 @@ const TableGridEditor: FC<Props> = ({
   const { meta, ui, vault } = useStore()
   const router = useRouter()
   const gridRef = useRef<SupabaseGridRef>(null)
-  const projectRef = ui.selectedProject?.ref
+  const { project } = useProjectContext()
```

Comment:
should we just use { ref } from useParams?

Reply:
ahh okay actually ignore this - i guess its consistency, i was searching for what other properties of `project` are you using for the context before i saw this one

Reply:
yeah this is mainly me trying to reduce the use of mobx as I go along